beam.cabal

```diff
@@ -2,7 +2,7 @@
 -- see http://haskell.org/cabal/users-guide/
 
 name:                beam
-version:             0.4.0.0
+version:             0.4.1.0
```

Comment:
I tried to follow PVP here. We didn't change the type signature of anything, and only added new functionality, so the minor version was bumped.